optuna/importance/__init__.py

```diff
@@ -0,0 +1,35 @@
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from optuna._experimental import experimental
+from optuna.importance._base import BaseImportanceEvaluator
+from optuna.importance._fanova import FanovaImportanceEvaluator
+from optuna.study import Study
+
+
+@experimental("1.3.0")
+def get_param_importances(
```

Comment:
Alternatives that might be more informative / better in some other sense

- `evaluate_param_importances`
- `evaluate_importances`
- `evaluate`

Other suggestions are highly welcomed.